docs/application/native/guides/ui/dali/progressbar-n.md

```diff
@@ -0,0 +1,120 @@
+[Creating a ProgressBar](#2)<br>
+[ProgressBar Properties](#3)<br>
+[Customizing the ProgressBar Appearance](#4)<br>
+
+## Overview
+
+The ProgressBar has two modes to represent the progress. The default mode is Determinate mode.
+
+- **Determinate mode** is to show a specific quantity of progress occurred. If you want to set the amount of progress, then set `PROGRESS_VALUE` property. If not supplied, the default value is 0.
+
+ *Figure: Determinate progress bar*
+ ![Determinate mode](./media/determinated_progress.png)
+
+
+- **Indeterminate mode** is to show the progress without a specific amount of progress indicated. If you want to indicate an indeterminate progress bar, then set `INDETERMINATE` enabled. You can also show an animation using `INDETERMINATE_VISUAL_ANIMATION` property.
+
+ *Figure: Indeterminate progress bar*
+ ![Indeterminate mode](./media/indeterminated_progress.png)
+
+<a name="1"></a>
+## ProgressBar events
+
+The following table lists the basic signal provided by the `Dali::Toolkit::ProgressBar` class.
+
+**Table: Dali::Toolkit::ProgressBar input signal**
+
+| Input signal              | Description                                 |
+| ------------------------- | ------------------------------------------- |
+| `ValueChangedSignal()`    | Emitted when the ProgressBar value changes. |
+
+<a name="2"></a>
+## Creating a ProgressBar
+
+The following basic example shows how to create a `Dali::Toolkit::ProgressBar` object:
+
+```
+ProgressBar progressBar = ProgressBar::New();
+progressBar.SetParentOrigin(ParentOrigin::TOP_CENTER);
+progressBar.SetAnchorPoint(AnchorPoint::TOP_CENTER);
+progressBar.SetResizePolicy(ResizePolicy::FILL_TO_PARENT, Dimension::WIDTH);
+progressBar.SetResizePolicy(ResizePolicy::USE_NATURAL_SIZE, Dimension::HEIGHT);
+progressBar.ValueChangedSignal().Connect( this, &ProgressBarExample::OnValueChanged );
+Stage::GetCurrent().Add( progressBar );
+```
+
+<a name="3"></a>
+## ProgressBar Properties
+
+You can modify the progress bar appearance and behavior through its properties.
+
+To change a property from its default value, use the `SetProperty()` function:
+
+```
+// Create a timer to update the progress of all progress bars
+mTimer = Timer::New( 50 );
+mTimer.TickSignal().Connect( this, &ProgressBarExample::OnTimerTick );
+mTimer.Start();
+
+...
+
+// To set the values, use SetProperty()
+bool OnTimerTick()
+{
+  mProgressValue += 0.01f; // float
+  mSecondaryProgressValue = mProgressValue + 0.1f; // float
+  progressBar.SetProperty( ProgressBar::Property::PROGRESS_VALUE, mProgressValue );
+  progressBar.SetProperty( ProgressBar::Property::SECONDARY_PROGRESS_VALUE, mSecondaryProgressValue );
+
+  // Only call again if progress has NOT got to the end
+  return ( mProgressValue < 1.0f );
+}
+```
+
+The following table lists the available progress bar properties.
+
+**Table: ProgressBar properties**
+
+| Property                   | Type    | Description                                   |
```

Comment:
In this table modify the description for all the rows as per edits given below.

This property stores the progress value of progress bar. It ranges between 0 and 1.
This property stores the secondary progress value of progress bar. It ranges between 0 and 1.
This property sets the progress bar in indeterminate state.
This property tracks the visual value of progress bar. It is a full progress area which appears behind PROGRESS_VISUAL.
This property stores the Progress Visual value of progress bar. The size of the progress visual is changed based on PROGRESS_VALUE.
This property stores Secondary Progress Visual of progress bar. The size of the secondary progress visual is changed based on SECONDARY_PROGRESS_VALUE.
This property stores Indeterminate Visual of progress bar.
This property stores transition data for indeterminate visual animation.
This property stores the label visual of progress bar.

Reply:
Changed the description similar to your comment.